### ISSUE TITLE
Feature idea: Display of List-Unsubscribe Header

### ISSUE THREAD

**Issue by kronthto** [axllent/mailpit#236]
It could be nice to check for existance of the combinations of these headers:
```
List-Unsubscribe: <mailto:listrequest@example.com?subject=unsubscribe>,
<https://example.com/unsubscribe.html?opaque=123456789>
List-Unsubscribe-Post: List-Unsubscribe=One-Click
```
And give feedback whether they are valid (e.g. not missing the < > signs) and render a preview of the Unsubscribe button mailbox providers sometimes show.

**Comment by axllent** [comment]
@kronthto The solution I have come up with (to be included in the next release) will display an `Unsubscribe` "link" next to the "From" line if the `List-Unsubscribe` header exists.  When clicked it will show the parsed (valid) email and/or http unsubscribe inks below the tags (hidden by default). If there are validation errors, these will be displayed as icons which, when clicked or hovered over will display a tooltip with the error.  If there are any errors then the `Unsubscribe` link will also display a red icon to alert you.

If the message also contains a `List-Unsubscribe-Post` when another icon is displayed with the tooltop of the header value. 

![Unsubscribe](https://github.com/axllent/mailpit/assets/1463435/8e451a2a-d977-4594-8b60-15ee6077c117)

Is this what you were after?

**Comment by axllent** [comment]
This new feature has just been released in v1.13.0.  I also added syntax validation with a number of checks for [valid](https://github.com/axllent/mailpit/blob/v1.13.0/internal/tools/tools_test.go#L76-L91) and [invalid](https://github.com/axllent/mailpit/blob/v1.13.0/internal/tools/tools_test.go#L93-L105)  `List-Unsubscribe` syntax (based on the information I studied). 

Please feel free to re-open this ticket if this does not work as you expected?

**Comment by kronthto** [comment]
Wow this is awesome, great work!

**Comment by brilik** [comment]
Hello! I have also this problem, I can't see link to unsubscribe in from section.
I run it in docker with axllent/mailpit:v1.10.3 image

<img width="1631" alt="image" src="https://github.com/user-attachments/assets/7c501f4d-0810-49f9-b60d-07c38e365516">

Can you help me, why I don't see this unsubscribe?

**Comment by axllent** [comment]
You don't see it because you are running a **very** old version of Mailpit. This functionality was released in V1.13.0. Please update your Mailpit.

**Comment by brilik** [comment]
> You don't see it because you are running a **very** old version of Mailpit. This functionality was released in V1.13.0. Please update your Mailpit.

I'm inattentive, but with version 1.13.3 is working correct! Thank you for quick answer! It

**Comment by axllent** [comment]
@brilik  You should really always use the latest release which is currently v1.21.5.
